docs/snippets/beyondtrust-values.yml

```diff
@@ -0,0 +1,8 @@
+replicaCount: 1
```

Comment:
Where is this used? Was it miscommited?

Reply:
This file should not have been versioned, it was removed from the PR

Reply:
Hello, @gusfcarvalho I removed that file from PR, this is needless.